core/src/mindustry/content/Blocks.java

```diff
@@ -1527,6 +1527,24 @@ public void drawBase(Tile tile){}
 
             consumes.powerCond(3f, (TractorBeamBuild e) -> e.target != null);
         }};
+        
+        clapper = new ItemTurret("clapper"){{
+            requirements(Category.turret, with(Items.plastanium, 230, Items.graphite, 150, Items.surgealloy, 130, Items.metaglass, 70));
+            ammo(
+            Items.graphite, plastBomb
```

Comment:
If it creates "plastanium bombs", shouldn't it [have the option to] consume plastanium?

Reply:
Oh, sorry, I just completely forgot to change the shell